code/game/jobs/job/civilian_chaplain.dm

```diff
@@ -6,304 +6,438 @@
+				J = "Coder"
+				deity_name = "The Coderbus"
+			if("42")
+				B.name = "The Hitchhiker's Guide to the Galaxy"
+				deity_name = "Arthur Dent" //Don't care
+			if("spook", "spooky", "boo", "ghost")
+				B.name = "The Spooky Spook" //SPOOK
+				J = "Ghost"
+				deity_name = "The Spook" //SPOOK
+				H.equip_or_collect(new /obj/item/clothing/head/pumpkinhead(H), slot_head)
+			if("medbay", "ride", "wild ride", "cryo")
+				B.name = "The Wild Ride"
+				if(H.gender == FEMALE)
+					J = "Nurse"
+				else
+					J = "Doctor"
+				//Give them basic medical garb
+				H.equip_or_collect(new /obj/item/clothing/head/surgery/blue(H), slot_head)
+				H.equip_or_collect(new /obj/item/clothing/mask/surgical(H), slot_wear_mask)
+				deity_name = "The Chief Medical Officer"
+			if("busta", "bustatime", "zas", "airflow", "hardcore", "hardcores")
+				B.name = "The Hardcores"
+				if(!(M_HARDCORE in H.mutations))
+					H.mutations.Add(M_HARDCORE)
+				J = "Atmospherics Technician"
+				deity_name = "Bustatime"
+			if("me", "i", "narcissism", "self importance", "selfishness")
+				B.name = "The Teachings of [H]" //Quite literally
+				J = "God"
+				deity_name = "[H]" //Very literally, too
+			if("alcohol", "booze", "beer", "wine", "ethanol", "c2h6o")
+				B.name = "The Drunken Ramblings"
+				J = "Drunkard"
+				deity_name = "Hic"
+			if("robust", "robustness", "strength")
+				B.name = "The Rules of Robustness"
+				J = "Robuster"
+				deity_name = "The Robust"
+			if("suicide", "death", "succumb")
+				B.name = "The Sweet Release of Death"
+				J = "Reaper"
+				deity_name = "The Grim Reaper"
+			if("communism", "socialism")
+				B.name = "The Communist Manifesto"
+				J = "Komrade"
+				deity_name = "Karl Max"
+				H.equip_or_collect(new /obj/item/clothing/head/ushanka(H), slot_head)
+			if("capitalism", "free market", "liberalism")
+				B.name = "The Free Market"
+				J = "Stockholder"
+				deity_name = "Adam Smith"
+				H.equip_or_collect(new /obj/item/clothing/head/that(H), slot_head)
+			if("freedom", "america", "muhrica", "usa")
+				B.name = "The Constitution"
+				J = "Senator"
+				deity_name = "George Washington"
+				H.equip_or_collect(new /obj/item/clothing/head/libertyhat(H), slot_head)
+			if("fascism", "nazi", "national socialism")
+				B.name = "Mein Kampf"
+				J = "Feldbischof" //No seriously, that's a thing, look it up
+				deity_name = "Adolf Hitler"
+				H.equip_or_collect(new /obj/item/clothing/head/naziofficer(H), slot_head)
+			if("security", "space law", "law", "nanotrasen", "centcomm")
+				B.name = "Space Law"
+				J = "Nanotrasen Officer"
+				deity_name = "Nanotrasen"
+				H.equip_or_collect(new /obj/item/clothing/head/centhat(H), slot_head)
+			if("syndicate", "traitor", "syndie", "syndies")
+				B.name = "The Syndicate Bundle"
+				J = "Syndicate Agent"
+				deity_name = "The Syndicate"
+				H.equip_or_collect(new /obj/item/clothing/head/syndicatefake(H), slot_head)
+			if("cult", "narsie", "nar'sie", "narnar")

```

Comment:
It's extremely obvious that a Cultist wouldn't have an actual "Cultist" job title, and anyone who attacks on sight when bashed with "The Arcane Tome" needs a foot up their ass, because making a fake Arcane Tome is Librarian 101